<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="13098" started-at="2019-10-08T17:10:00 PDT" finished-at="2019-10-08T17:10:13 PDT">
    <groups>
    </groups>
    <test name="Default test" duration-ms="13098" started-at="2019-10-08T17:10:00 PDT" finished-at="2019-10-08T17:10:13 PDT">
      <class name="com.test.dataprovider.TestNG9ParameterizedExpresssRegTest">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.test.dataprovider.TestNG9ParameterizedExpresssRegTest@1877ab81]" name="setUp" is-config="true" duration-ms="10879" started-at="2019-10-08T17:10:00 PDT" finished-at="2019-10-08T17:10:11 PDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="enterRegInfo()[pri:0, instance:com.test.dataprovider.TestNG9ParameterizedExpresssRegTest@1877ab81]" name="enterRegInfo" duration-ms="2172" started-at="2019-10-08T17:10:11 PDT" finished-at="2019-10-08T17:10:13 PDT">
          <exception class="java.lang.IndexOutOfBoundsException">
            <message>
              <![CDATA[Index: 5, Size: 5]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.IndexOutOfBoundsException: Index: 5, Size: 5
at java.util.ArrayList.rangeCheck(ArrayList.java:653)
at java.util.ArrayList.get(ArrayList.java:429)
at com.test.dataprovider.TestNG9ParameterizedExpresssRegTest.enterRegInfo(TestNG9ParameterizedExpresssRegTest.java:39)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:497)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1249)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.IndexOutOfBoundsException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- enterRegInfo -->
      </class> <!-- com.test.dataprovider.TestNG9ParameterizedExpresssRegTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
